storybookjs/storybook
# Add radio buttons knob type #3872

Issue:
#3872 

## What I did
I created a radio button knob type

## How to test
I created unit tests for the radio button knob type. The first test checks whether or not the radio button 
correctly renders the labels appropriately. The second test ensures that the values set on the radio buttons are accurate. The last test checks for the correct radio button being marked as checked.

Is this testable with Jest or Chromatic screenshots? Tested
Does this need a new example in the kitchen sink apps? An example radio button knob type was added to the react kitchen sink app.
Does this need an update to the documentation? Information about the radio button addon knob type was added to the documentation in the README.md file for the knobs. 

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
